riot/lol/match.go

```diff
@@ -111,41 +106,17 @@ func (m *MatchClient) ListStream(accountID string) <-chan MatchStreamValue {
 
 // GetTimeline returns the timeline for the given match
 // NOTE: timelines are not available for every match
+// TODO: update to v5 when struct is documented
```

Comment:
This was too much data for me to try to do by hand; i put a pin in it untill they release the official documentation for the returned data